packages/legend-application-studio/src/stores/EditorGraphState.ts

```diff
@@ -127,28 +138,59 @@ export class EditorGraphState {
 
   constructor(editorStore: EditorStore) {
     makeAutoObservable(this, {
+      mostRecentFormModeCompilationGraphHash: observable,
+      isStaleTextWarnings: observable,
       editorStore: false,
       graphGenerationState: false,
+      warnings: true,
       getPackageableElementType: false,
-      hasCompilationError: computed,
       clearCompilationError: action,
+      setWarnings: action,
+      setIsStaleTextWarnings: action,
+      setMostRecentFormModeCompilationGraphHash: action,
+      hasCompilationError: computed,
+      currentGraphHash: computed,
```

Comment:
don't need this, just call `return this.editorStore.changeDetectionState.graphHash;` directly